src/app/shared/styles/_global.scss

```diff
@@ -0,0 +1,96 @@
+@import "./color";
+@import "./typography";
+@import "./template";
+
+*:not(form).ng-invalid.ng-touched input {
+  border-color: red;
+}
+
+.form {
+  button[type="submit"] {
+    margin-top: 40px;
```

Comment:
margin-top is bad practice in most cases

Reply:
@dmitry-darly-solutions 
Fixed.

Reply:
I tried to found how select element before last without requiring create additional class and not found way. Here it will use only to simplify styling.